scss/_variables.scss

```diff
@@ -642,7 +642,8 @@ $aspect-ratios: (
   "1x1": 100%,
   "4x3": calc(3 / 4 * 100%),
   "16x9": calc(9 / 16 * 100%),
-  "21x9": calc(9 / 21 * 100%)
+  "21x9": calc(9 / 21 * 100%),
+  "9x16": calc(16 / 9 * 100%),
```

Comment:
* Extra `,` to remove
* A "Boosted mod" comment is needed here

Reply:
Done.